mgmt/RecordsConfig.cc

```diff
@@ -1449,6 +1449,8 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.allocator.hugepages", RECD_INT, "0", RECU_RESTART_TS, RR_NULL, RECC_NULL, "[0-1]", RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.allocator.dontdump_iobuffers", RECD_INT, "0", RECU_RESTART_TS, RR_NULL, RECC_NULL, "[0-1]", RECA_NULL}
```

Comment:
I'm fine with it being backported if @bryancall is. Though, I don't think we absolutely have to. In 7.0.x it's on by default now, you just can't disable it.

I'm considering backporting this to 6.2.x as well due to the trouble it causes on older kernels.